### PR TITLE
5B updates

### PR DESCRIPTION
**Pull request by HongGit**
Update 5B

### CODE REVIEW

File: 3.5-windowsservercore-1803/runtime/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# escape=`
+
+FROM microsoft/dotnet-framework:3.5-windowsservercore-1803
```
Comment (MichaelSimons) [inline 3.5-windowsservercore-1803/runtime/Dockerfile]: This should be using the `runtime` tag - `3.5-runtime-windowsservercore-1803`

File: 4.7.2-windowsservercore-1803/runtime/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# escape=`
+
+FROM microsoft/dotnet-framework:4.7.2-runtime-windowsservercore-1803
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV COMPLUS_NGenProtectedProcess_FeatureEnabled 0
```
Comment (MichaelSimons) [inline 4.7.2-windowsservercore-1803/runtime/Dockerfile]: This shouldn't be needed anymore as it is part of the base image.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
-﻿# Supported Windows Server, version 1709 amd64 tags
+﻿# Supported Windows Server, version 1803 amd64 tags
```
Comment (MichaelSimons) [inline README.md]: We've been dropping the term 'Supported' across our .NET Docker readmes as it is rather meaningless.  Dropping it makes it a little easier to quickly scan and differentiate the sections.

Comment (HongGit) [reply README.md]: oops, I merged too quick before I see this comment.

I will create a new PR.

**Pull request merged**